index.html

```diff
@@ -0,0 +1,312 @@
+        </div>
+        <p class="feature-card__description">the quick fox jumps over the lazy
+          dog the quick fox jumps over the
+          lazy dog</p>
+      </div>
+    </div>
+  </section>
+
+  <section class="services">
+    <div class="section-text">
+      <h1 class="section-text__heading">We are providing best
+        business service.</h1>
+      <p class="description">We know how large objects will act, but things
+        on a
+        small scale just do not act that way.</p>
+    </div>
+    <div><img class="services__image" src="./assets/images/card-item.png" alt="video image"></div>
+    <div class="services__image-background">
+    </div>
+
+  </section>
+
+  <section class="people-section">
+    <div class="section-text">
+      <h1 class="section-text__heading">What Clients Say</h1>
+      <p class="description">Problems trying to resolve the conflict between
+        the two major realms of Classical physics: Newtonian mechanics</p>
+    </div>
+    <div class="people-section__card-container">
+      <div class="testimony-card">
+        <img src="./assets/icons/stars.svg" alt="rating stars" class="testimony-card__ratings">
+        <p class="description">Slate helps you see how many
+          more days you need to work to
+          reach your financial goal.</p>
+        <div class="testimony-card__footer">
+          <img src="./assets/images/Ellipse 2.png" alt="person" class="testimony-card__user-image">
+          <div class="testimony-card__user-details">
+            <a href="#" class="headline headline--yellow">Regina Miles</a>
+            <h6 class="headline headline--dark">Designer</h6>
+          </div>
+        </div>
+      </div>
+      <div class="testimony-card">
+        <img src="./assets/icons/stars.svg" alt="rating stars" class="testimony-card__ratings">
+        <p class="description">Slate helps you see how many
+          more days you need to work to
+          reach your financial goal.</p>
+        <div class="testimony-card__footer">
+          <img src="./assets/images/Ellipse 2.png" alt="person" class="testimony-card__user-image">
+          <div class="testimony-card__user-details">
+            <a href="#" class="headline headline--yellow">Regina Miles</a>
+            <h6 class="headline headline--dark">Designer</h6>
+          </div>
+        </div>
+      </div>
+      <div class="testimony-card">
+        <img src="./assets/icons/stars.svg" alt="rating stars" class="testimony-card__ratings">
+        <p class="description">Slate helps you see how many
+          more days you need to work to
+          reach your financial goal.</p>
+        <div class="testimony-card__footer">
+          <img src="./assets/images/Ellipse 2.png" alt="person" class="testimony-card__user-image">
+          <div class="testimony-card__user-details">
+            <a href="#" class="headline headline--yellow">Regina Miles</a>
```

Comment:
`headline--yellow` isn't a good BEM name. Naming your classes or modifiers or elements after their CSS properties isn't recommended in BEM. Something like `headline--colored` so you don't have to change everywhere (CSS and HTML) whenever the change in color happens.